webpack.config.js

```diff
@@ -4,6 +4,8 @@ var nodeExternals = require('webpack-node-externals');
 var Fs = require('fs')
 var nodeModules = {}
 var ExtractTextPlugin = require('extract-text-webpack-plugin');
+var autoprefixer = require('autoprefixer');
+var precss = require('precss');
```

Comment:
Also, we have to require `autoprefixer` because we are using it in the `postcss` function:
```
postcss: function () {
    return [autoprefixer({browsers: ['last 2 versions']})];
  },
```